### ISSUE TITLE
SimpleAutogamestart:  Loads character named odev

### ISSUE THREAD

**Issue by Boatsandlogs** [abearcodes/Valheim#14]
I put in my character name as it appears in the save directory, and it loads my configured world.  However it loads a character named ODEV every time no matter if I spell my name Nicholas or nicholas in config

**Comment by abearcodes** [comment]
Sorry, I don't have a lot of time to look into it right now and the other mods have more priority. I will queue it up for when I have more time available and the other mods require less attention

**Comment by Razinell** [comment]
I edited the config file with my correct world name and character name and the game did auto-start, but into a new game with a new character. Scared the crap out of me, thinking I lost all my dozens of hours of progress in the game. But my savegame is okay. But I have disabled this mod for now.

**Comment by abearcodes** [comment]
Yeah... there’s a discrepancy between profile names and character names so that can happen.. I initially developed this mod only to get faster into the game when needing to restart during development so it’s lower on my priority list... but hey, it’s either this or the new crafting tab rework for Simple Recycling.. I would say it’s warranted neglect 😄 

Will update this ticket once I fix the issue.
